src/app/containers/ATIAnalytics/params/article/index.jsx

```diff
@@ -1,44 +1,13 @@
 import { useContext } from 'react';
-import atiPageViewParams from '../../atiUrl';
 import { ServiceContext } from '#contexts/ServiceContext';
 import { RequestContext } from '#contexts/RequestContext';
-
-import { getPublishedDatetime } from '#lib/analyticsUtils';
-import {
-  getLanguage,
-  getContentId,
-  getPageIdentifier,
-  getPromoHeadline,
-  getThingAttributes,
-} from '#lib/analyticsUtils/article';
+import { buildArticleATIUrl } from './buildParams';
 
 const ArticleAtiParams = articleData => {
```

Comment:
I'm confused about this file. Is this actually a React component? If so then where are the prop types?

Reply:
I've taken it out. See [my comment](https://github.com/bbc/simorgh/pull/3559#discussion_r328943230)